examples/hyperpod-eks/train.yaml

```diff
@@ -0,0 +1,30 @@
+resources:
```

Comment:
Maybe we can reference to `examples/distributed-pytorch/train.yaml` instead of duplicating the file? Not feeling strongly

Reply:
Wanted to make this example folder self-contained and we can add more EFA related instructions here in the future as well : )